### PR TITLE
ECS: Impose hard limits for memory and cpu

### PR DESCRIPTION
**Pull request by mirikl**
## Description
Impose hard limits for memory and cpu in gprofiler ECS task definition

### CODE REVIEW

File: deploy/ecs/gprofiler_task_definition.json [hunk 1]
```diff
@@ -6,9 +6,7 @@
         {
             "name": "granulate-gprofiler",
             "image": "index.docker.io/granulate/gprofiler:latest",
-            "memory": "1024",
```
Comment (Jongy) [inline deploy/ecs/gprofiler_task_definition.json:9]: Is that the case for memory as well? I was under the impression that `memory` is respected under the container, but `cpu` is not.

File: deploy/ecs/gprofiler_task_definition.json [hunk 2]
```diff
@@ -6,9 +6,7 @@
...
             "memoryReservation": "256",
-            "cpu": "100",
```
Comment (Jongy) [inline deploy/ecs/gprofiler_task_definition.json:11]: Value is still required here, to serve as the "requests" for CPU. Keep it

File: deploy/ecs/gprofiler_task_definition.json (hunk 1 above)
Comment (mirikl) [reply deploy/ecs/gprofiler_task_definition.json]: You're right I'll keep it in container level.

**Review by Jongy — APPROVED** [review]

File: deploy/ecs/gprofiler_task_definition.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
             "privileged": true
         }
     ],
+    "cpu": "1024",
```
Comment (Jongy) [inline deploy/ecs/gprofiler_task_definition.json:22]: Actually, let's use 512 to match k8s 500

**Review by Jongy — CHANGES_REQUESTED** [review]
.

**Review by Jongy — APPROVED** [review]

**Pull request merged**